test/testcases/auth/put.password.test.ts

```diff
@@ -374,6 +374,13 @@ describe('PUT /auth/password - Change password', () => {
       .send({currentPassword: 'Password13!'});
     expect(response.status).toBe(400);
 
+    // Password Change request
+    response = await request(testEnv.expressServer.app)
+      .put('/auth/password')
+      .set('Cookie', [`X-REFRESH-TOKEN=${refreshToken}`])
+      .send({});
+    expect(response.status).toBe(400);
+
```

Comment:
no 'expect(response.body.error).toBe('Bad Request');' here?

Reply:
Since response's status is 400 (HTTP status code: 400 Bad request), I think it is OK to be used.

Reply:
Its not perfect but it is still okay.